### ISSUE TITLE
Implement a tutorial for new users to the site.

### ISSUE THREAD

**Issue by jdhtwo** [mattcroce/HSver1#30]
Pop-up windows for "welcome, here's how this works". Maybe only use this for the "Get Started"? Not sure.

**Comment by jdhtwo** [comment]
For example, I recently signed up for MailChimp. Check out these nice little popups. Just .divs, baby!

![1](https://cloud.githubusercontent.com/assets/2292853/7237155/aeaa4d88-e767-11e4-97ff-929e30bb0007.png)
![2](https://cloud.githubusercontent.com/assets/2292853/7237157/b085a0da-e767-11e4-90b4-a1a2f185086d.png)

**Comment by mattcroce** [comment]
I like this idea...need to implement some sort of user tracking (ip address, email address?) in order to ensure that someone doesn't get bombarded with this every time they visit the site - only the first visit or if they haven't been to the site in 6 months or something like that.

**Comment by jdhtwo** [comment]
Agreed. I think my steps would be 1) to make the pop up appear at all (thus, every time) and then 2) get it to only appear based on some criteria (as you list).

**Comment by mattcroce** [comment]
Alternative option might be an 'entry portal' or something. First page is really basic with two options - take a tour or skip to main site. It's easy (AKA, I think I know how to do it) to make a pop-up happen on a click event. I'm sure it's also easy to make it happen for a new visitor without requiring a click, but I just don't know off the top of my head. Enter the Google.
